MODULE.bazel

```diff
@@ -31,9 +64,15 @@ use_repo(remote_android_extensions, "android_gmaven_r8", "android_tools")
 # TODO(bencodes) A bunch of these dependencies need to be marked as dev_dependencies but before we can do that
 # we need to sort out a few cases around how these rules are consumed in various ways.
 
-bazel_dep(name = "rules_jvm_external", version = "5.3")
+bazel_dep(
```

Comment:
Can you undo the formatting changes in `bazel_dep`?

Reply:
Kind of just running buildifier here -- I'm pretty unopinionated on the formatting.

Do we need to tell buildifier to bugger off in bzlmod?

Reply:
Done.